kedro-airflow/pyproject.toml

```diff
@@ -24,7 +24,7 @@ Tracker = "https://github.com/kedro-org/kedro-plugins/issues"
 
 [project.optional-dependencies]
 test = [
-    "apache-airflow<3.0",
+    "apache-airflow<2.7.0",
```

Comment:
+1 but can we add a comment with an explanation? so we remember when we get to fix it

Reply:
Done!